microsoft-edge/devtools-guide-chromium/resources/index.md

```diff
@@ -20,139 +20,175 @@ ms.date: 05/04/2021
+
+   Type filename:
+
+   ![Typing part of a filename in the Open File list of the Command Menu](../media/resources-command-menu-file-search.msft.png)
 
 
 <!-- ====================================================================== -->
-## Open resources in the Network tool
+## Browse resource files in the Network tool
 
-See [Inspect the details of a resource](../network/index.md#inspect-the-details-of-the-resource) in _Inspect network activity_.
-<!--why?-->
+In the **Network** tool, you can inspect the resource files that make up the current webpage, such as `.html`, `.css`, `.js`, and image files.  To learn how to get the details for a specific resource, see [Inspect the details of the resource](../network/index.md#inspect-the-details-of-the-resource) in _Inspect network activity_.
 
-<!-- need lead-in -->
+When you go to the **Network** tool and then refresh the page, all the webpage's resource files are listed in the **Network** tool, one row per resource file:
 
-![Inspect a resource in the Network tool.](../media/resources-network-response.msft.png)
+![Webpage resource files listed in the Network Log, one resource file per row](../media/resources-network-resources.msft.png)
+
+When **All** is selected as the resources filter, all the retrieved resource files for the current webpage are listed, in the lower left:
 
-### Reveal resources in the Network tool from other panels
+![Inspect a resource in the Network tool](../media/resources-network-response.msft.png)
 
-The next section, [Browse resources in the Network panel](#browse-resources-in-the-network-panel), shows how to view resources from various parts of the DevTools UI.  To inspect a resource in the **Network** tool, right-click the resource and select **Reveal in Network panel**.
+The above screenshot shows `getstarted.html`, though a folder is selected in the lower left.
+
+To learn how to show the network activity that's generated by the page resources, see [Log network activity](../network/index.md#log-network-activity) in the article  _Inspect network activity_.
+
+![Inspect a resource in the Network tool.](../media/resources-network-response.msft.png)
 
-![Revealing resources in the Network panel.](../media/resources-sources-page-reveal-in-network-panel.msft.png)
 
 
 <!-- ====================================================================== -->
-## Browse resources in the Network panel
+## Show resource files in the Network tool from other tools
+
+In various tools that list resource files for the webpage, to inspect a resource file in the **Network** tool, right-click the resource file and then select **Reveal in Network panel**.  You may need to first refresh the webpage while the **Network** tool is open.
+
+For example, to go from the **Page** tab's resource list in the **Sources** tool to the **Network** tool:
+
+1. In a new window or tab, go to [https://microsoftedge.github.io/Demos/demo-to-do/](https://microsoftedge.github.io/Demos/demo-to-do/).
```

Comment:
```suggestion
1. In a new window or tab, go to a webpage, such as the [Inspect Network Activity Demo](https://microsoftedge.github.io/Demos/network-tutorial/).  (The source files for that webpage are at [MicrosoftEdge / Demos > network-tutorial](https://github.com/MicrosoftEdge/Demos/tree/main/network-tutorial).)
```

Reply:
Did you mean to apply this change?  It wasn't applied.  This change fixes a disparity, this line says demo-to-do yet the screenshot shows a different demo.

Reply:
missed, added locally